wincent/liferay-portal
# LPS-113561 Avoid use of Liferay.provide in sync-web

As part of [LPS-113561](https://issues.liferay.com/browse/LPS-113561) we
eventually want to deprecate the `Liferay.provide` function.
Additionally, the `Liferay.Util.getWindow` and `Liferay.Util.openWindow`
functions have been replaced with the `Liferay.Util.getOpenener` and
`Liferay.Util.openModal` functions.

To test this change:

- From the "Global Menu" select the "Control Panel" tab
- Click on "Sync Connector Admin"
- Select the "Sites" tab
- In the sites list, click the kebab menu and select "Default File
  Permissions"
- Select one of the permissions in the modal window

As a result, the new permissions should be set for the site and no JS
errors should appear in the browser's console.
